nightly-run.sh

```diff
@@ -10,6 +10,7 @@ SEARCH_NODE=$(/usr/local/bin/govuk_node_list -c search --single-node)
 if [[ -z $SKIP_TRAFFIC_LOAD ]]; then
   if [ \! -d ENV ]; then virtualenv -p /usr/bin/python3 ENV; fi
   . ENV/bin/activate
+  pip install --upgrade setuptools
```

Comment:
We have an ancient version of setuptools because the ci-agents are using Python 3.4 - which was released in 2014 and is end of life: https://www.python.org/downloads/release/python-340/.  I think this will work, but it might be better to look into updating Python.

Reply:
Yeah, I think this script runs on Jenkins in the given environment, which has python 3.4.

Shall we merge this as a quickfix so popularity starts working again?